src/main/java/solvd/ermakovich/zt/kafka/srteams/config/health/indicator/HealthIndicatorPredicate.java

```diff
@@ -0,0 +1,42 @@
+package solvd.ermakovich.zt.kafka.srteams.config.health.indicator;
+
+import java.util.List;
+import java.util.function.BiPredicate;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.streams.kstream.Predicate;
+import org.springframework.stereotype.Service;
+import solvd.ermakovich.zt.domain.health.indicators.DefaultHealthIndicators;
+import solvd.ermakovich.zt.domain.health.indicators.HealthIndicatorsMessage;
+import solvd.ermakovich.zt.service.DefaultHealthIndicatorService;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class HealthIndicatorPredicate
```

Comment:
@kermakovich
this class has such a vague name, maybe we should rename it
also, `-predicate` suffix can be eliminated here